Samples/Samples.iOS/Info.plist

```diff
@@ -55,6 +55,10 @@
 	<string>1.0</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Access to your location is required for cool things to happen!</string>
+    <key>NSCalendarsUsageDescription</key>
+    <string>Access to view and create calendar events</string>
+    <key>NSRemindersUsageDescription</key>
```

Comment:
remove NSRemindersUsageDescription and test

Reply:
Addressed in: https://github.com/builttoroam/Essentials/pull/43